.changeset/five-berries-rest.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": major
+---
+
+Remove constellation commands and binding, please migrate to Workers AI, learn more here https://developers.cloudflare.com/workers-ai/.
```

Comment:
```suggestion
Remove experimental/beta constellation commands and binding, please migrate to Workers AI, learn more here https://developers.cloudflare.com/workers-ai/.

This is not deemed a breaking change to Wrangler since these commands were never generally available.
```